docs/source/executing-operations/subscription-support.mdx

```diff
@@ -1,17 +1,19 @@
 ---
-title: GraphQL subscriptions with a self-hosted router
-description: Real-time data with GraphOS Enterprise
+title: Configure subscription support
+subtitle: Enable clients to receive real-time updates whenever new data becomes available
+description: Configure your router to support GraphQL subscriptions, enabling clients to receive real-time updates via WebSocket or HTTP callbacks.
```

Comment:
Keeping the title `GraphQL subscriptions with a self-hosted router` maintains the parallelism with the title `GraphQL subscriptions with a cloud supergraph` of https://www.apollographql.com/docs/graphos/operations/subscriptions. Or we can edit both?

```suggestion
title: GraphQL subscriptions with a self-hosted router
subtitle: Enable clients to receive real-time updates whenever new data becomes available
description: Configure your router to support GraphQL subscriptions, enabling clients to receive real-time updates via WebSockets or HTTP callbacks.
```

Reply:
This page isn't limited to information for self-hosted routers, hence the title change.

Reply:
This page was meant for self-hosted, and even with the title change a reader right now would probably assume it's limited to self-hosted because it's under `Self-Hosted Routers`.

This is a jumping off point to start revamping the IA of router docs and define the path that consolidates common, self-hosted, and cloud topics.